2022 - Legiones Astartes.cat

```diff
@@ -24393,26 +24393,70 @@ A Perfect Fury: If a model or models with the Skill Unmatched special rule selec
+          <description>A unit that contains only models with this special rule suffers no penalty for moving or charging through Difficult Terrain.</description>
+        </rule>
+      </rules>
       <categoryLinks>
         <categoryLink id="c543-976f-ef9d-d41e" name="Legiones Astartes" hidden="false" targetId="11f2-472f-c1d1-9ae9" primary="false"/>
         <categoryLink id="9152-5697-6c00-ca89" name="Dreadnought Unit-type:" hidden="false" targetId="4280-4963-02b5-e31d" primary="false"/>
         <categoryLink id="7423-159b-9fbd-f718" name="Heavy" hidden="false" targetId="9231-183c-b97b-63f9" primary="false"/>
       </categoryLinks>
       <selectionEntries>
-        <selectionEntry id="ac2d-77a1-ecd7-a555" name="Leviathan" hidden="false" collective="false" import="true" type="model">
+        <selectionEntry id="ac2d-77a1-ecd7-a555" name="Leviathan Dreadnought" hidden="false" collective="false" import="true" type="model">
           <constraints>
             <constraint field="selections" scope="parent" value="1.0" percentValue="false" shared="true" includeChildSelections="false" includeChildForces="false" id="38cd-dc83-4d9f-ecac" type="min"/>
             <constraint field="selections" scope="parent" value="3.0" percentValue="false" shared="true" includeChildSelections="false" includeChildForces="false" id="3574-849a-7cfe-73f4" type="max"/>
           </constraints>
+          <profiles>
+            <profile id="6bc5-3dcd-bcdf-ff32" name="Leviathan Dreadnought" hidden="false" typeId="4bb2-cb95-e6c8-5a21" typeName="Unit">
+              <characteristics>
+                <characteristic name="Unit Type" typeId="ddd7-6f5c-a939-b69e">Dreadnought (Heavy)</characteristic>
+                <characteristic name="Move" typeId="893e-2d76-8f04-44e5">6</characteristic>
+                <characteristic name="WS" typeId="cc42-7ed5-7092-5c84">5</characteristic>
+                <characteristic name="BS" typeId="74ae-c840-0036-d244">5</characteristic>
+                <characteristic name="S" typeId="e478-41d4-a092-48a8">8</characteristic>
+                <characteristic name="T" typeId="c32b-5fdd-3fbe-9b1f">8</characteristic>
+                <characteristic name="W" typeId="57ee-1126-32a9-5672">7</characteristic>
+                <characteristic name="I" typeId="62d3-22d7-2d49-52dc">4</characteristic>
+                <characteristic name="A" typeId="f111-2ce5-dd12-d6b0">5</characteristic>
+                <characteristic name="Ld" typeId="e8a6-1da9-d384-8727">9</characteristic>
+                <characteristic name="Save" typeId="e593-6b3c-f169-04f0">2</characteristic>
+              </characteristics>
+            </profile>
+          </profiles>
+          <selectionEntries>
+            <selectionEntry id="a3ad-3b70-433b-b9fa" name="Atomantic Deflector" hidden="false" collective="false" import="true" type="upgrade">
+              <profiles>
```

Comment:
Automantic deflector is not an option. This should either be an info link to wargear or a link to a shared selection entry with min1 max1.